packages/clay-css/src/scss/variables/_multi-step-nav.scss

```diff
@@ -80,7 +80,7 @@ $multi-step-icon-active-bg-position: center !default;
 $multi-step-icon-active-bg-size: $multi-step-icon-font-size !default;
 $multi-step-divider-active-bg: $multi-step-divider-bg !default;
 
-$multi-step-icon-disabled-bg: $gray-100 !default;
+$multi-step-icon-disabled-bg: $gray-300 !default;
```

Comment:
Don't need to do this in `src/scss/atlas/variables/_multi-step-nav.scss`. It inherits from `$multi-step-icon-bg`.